docs/data-structures.md

```diff
@@ -0,0 +1,249 @@
+  ],
+  "args": [
+    "3044022038f282cffdd26e2a050d7779ddc29be81a7e2f8a73706d2b7a6fde8a78e950ee0220538657b4c01be3e77827a82e92d33a923e864c55b88fd18cd5e5b25597432e9b",
+    "1"
+  ]
+}
+```
+
+
+
+### Description
+
+| Name          | Type    | Description                                                  |
+| :------------ | :------ | :----------------------------------------------------------- |
+| `version`     | uint8   | **The version of the script.** It‘s used to distinguish transactions when there's a fork happened to the blockchain system. |
+| `binary`      | Bytes   | **ELF formatted binary that contains an RISC-V based script.** This part of data is loaded into an CKB-VM instance when they are specified upon the transaction verification. |
+| `reference`   | Bytes   | **The hash of the script that is referred by this script.** It is possible to refer the script in another cell on-chain as the binary code in this script, instead of entering the binary directly into the script. **Notice:** This is part only works when the `binary` field is empty. |
+| `args`        | [Bytes] | **An array of arguments for the script.** The arguments here are imported into the CKB-VM instance as input arguments for the scripts. This part is NOT used when calculating the hash of the script. |
+| `signed_args` | [Bytes] | **An array of signed arguments for the script.** The arguments with signatures. The `signed_args` and the `args` will be connected into a single vector, and imported into the CKB-VM instance as input arguments. |
+
+
+
+More information about Script can be [here](https://github.com/nervosnetwork/ckb-demo-ruby-sdk/blob/27669cd6b4f56f8977b725fc0e6582b288ab2b82/docs/how-to-write-contracts.md#script-model).
+
+
+
+## Transaction
+
+### Example
+
+```json
+{
+    "deps": [],
+    "hash": "0x4707810253259258e3091934dc5b543403c10cc899859e077fe26067f8d52dc0",
+    "inputs": [
+      {
+        "previous_output": {
+          "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+          "index": 4294967295
+        },
+        "unlock": {
+          "args": [],
+          "binary": "0x0d00000000000000",
+          "reference": null,
+          "signed_args": [],
+          "version": 0
+        }
+      }
+    ],
+    "outputs": [
+      {
+        "capacity": 5000000,
+        "data": "0x",
+        "lock": "0xa58a960b28d6e283546e38740e80142da94f88e88d5114d8dc91312b8da4765a",
+        "type": null
+      }
+    ],
+    "version": 0
+}
+```
+
+### Description
+
+#### Transaction
+
+| Name              | Type                             | Description                                                  |
+| ----------------- | -------------------------------- | ------------------------------------------------------------ |
+| `version`         | uint32                           | **The version of the transaction.** It‘s used to distinguish transactions when there's a fork happened to the blockchain system. |
```

Comment:
```suggestion
| `version`         | uint32                           | **The version of the transaction.** It‘s used to distinguish transactions when there's a fork happened to the blockchain. |
```

Reply:
"blockchain" is just a data structure. 

"blockchain system" is the system that utilize a blockchain data structure.